### ISSUE TITLE
Error indicator on span in /Traces is hard to see in dark mode

### ISSUE THREAD

**Issue by tlmii** [dotnet/aspire#690]
The contrast between the red and the dark background doesn't meet accessibility standards, which might be ok if we could rely on the shape as well. But the outer shape is always a circle and the inner shape is hard to tell also, in part because of the size and in part because of the contrast:

![image](https://github.com/dotnet/aspire/assets/9613109/80ad73cb-96fc-4db1-a493-9fa954229b7b)

It's a little better in light mode since the red/white contrast is better, but the shape of the indicator in the circle is still hard to see unless it is big enough:

![image](https://github.com/dotnet/aspire/assets/9613109/9795913f-7032-427d-b272-9a984311ddf1)

This isn't _just_ an a11y issue since it is hard to tell what the icon is at 100% scaling, but accessibility is definitely a part of how we'd address it.

A few thoughts:
1. We should see if we can adjust the icon in some way to make it easier to decipher (use regular instead of filled? use Size16 instead?). If not, we should consider if there's a different way to display that information
2. These individual blocks _could_ be light-colored even if in dark mode (see screenshot below) if we wanted to keep the error red the same. But we should also see about trying to make an error red color that works better on the dark background across the site.

![image](https://github.com/dotnet/aspire/assets/9613109/24f52ac5-fd45-41ba-a487-c794b8271e77)

**Comment by JamesNK** [comment]
Just make the `--error` var red slightly lighter in light mode.

**Comment by adamint** [comment]
Option 3 is overriding the error color to a shade that we know is accessible against both backgrounds. I prefer this
